iroha/config.json

```diff
@@ -43,6 +43,9 @@
     "digest_function": "ed25519",
     "payload": "9ac47abf59b356e0bd7dcbbbb4dec080e302156a48ca907e47cb6aea1d32719e7233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0"
   },
+  "GENESIS_CONFIGURATION": {
+    "GENESIS_FILE_PATH": "tests/genesis.json"
+  },
```

Comment:
We already pass genesis file path in the command line through `./iroha --genesis="tests/genesis.json"`, so why should we need this field here?

Reply:
Sorry, I forgot to remove this.